dotnet/src/SemanticKernel/CoreSkills/TextMemorySkill.cs

```diff
@@ -43,9 +43,22 @@ public class TextMemorySkill
     public const string LimitParam = "limit";
 
     private const string DefaultCollection = "generic";
-    private const string DefaultRelevance = "0.75";
+    private const string DefaultRelevance = "0.0";
```

Comment:
this seems a pretty big change, how so?

Reply:
Guidance was from @awharrison-28 in above comments. The current value is very much dependent on the ADA models, which may not be the default for users.